tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -522,3 +522,189 @@ Feature: collaboration (wopi)
       | app-endpoint                                              |
       | /app/open-with-web?file_id=<<FILEID>>&app_name=FakeOffice |
       | /app/open-with-web?file_id=<<FILEID>>                     |
+
+
+  Scenario: create a text file using wopi endpoint in Personal space
+    Given user "Alice" has created folder "testFolder"
+    When user "Alice" creates a file "testfile.txt" inside folder "testFolder" in space "Personal" using wopi endpoint
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "file_id"
+        ],
+        "properties": {
+          "file_id": {
+            "type": "string",
+            "pattern": "^%file_id_pattern%$"
+          }
+        }
+      }
+      """
+    And as "Alice" file "testFolder/testfile.txt" should exist
+
+
+  Scenario Outline: sharee with permission Editor/Uploader creates text file inside shared folder
+    Given user "Alice" has created folder "testFolder"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | testFolder         |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    When user "Brian" creates a file "testFile.txt" inside folder "testFolder" in space "Shares" using wopi endpoint
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "file_id"
+        ],
+        "properties": {
+          "file_id": {
+            "type": "string",
+            "pattern": "^%file_id_pattern%$"
+          }
+        }
+      }
+      """
+    And as "Alice" file "testFolder/testFile.txt" should exist
+    And as "Brian" file "Shares/testFolder/testFile.txt" should exist
+    Examples:
+      | permissions-role |
+      | Editor           |
+      | Uploader         |
+
+
+  Scenario: sharee with Viewer tries to create text file inside shared folder
```

Comment:
What is viewer? plz clarify that, I see some inconsistencies in **role and permission** word uses. Please enlight me on that too

Reply:
just missed the wording.